docs/designers-developers/developers/data/data-core-editor.md

```diff
@@ -1,370 +1,701 @@
+
+Returns true if a previous post save was attempted but failed, or false
 otherwise.
 
-*Parameters*
+_Parameters_
 
- * state: Global application state.
+-   _state_ `Object`: Global application state.
 
-*Returns*
+_Returns_
 
-Whether redo history exists.
+-   `boolean`: Whether the post save failed.
 
-### isEditedPostNew
+<a name="didPostSaveRequestSucceed" href="#didPostSaveRequestSucceed">#</a> **didPostSaveRequestSucceed**
 
-Returns true if the currently edited post is yet to be saved, or false if
-the post has been saved.
+Returns true if a previous post save was attempted successfully, or false
+otherwise.
 
-*Parameters*
+_Parameters_
 
- * state: Global application state.
+-   _state_ `Object`: Global application state.
 
-*Returns*
+_Returns_
 
-Whether the post is new.
+-   `boolean`: Whether the post was saved successfully.
 
-### hasChangedContent
+<a name="getActivePostLock" href="#getActivePostLock">#</a> **getActivePostLock**
 
-Returns true if content includes unsaved changes, or false otherwise.
+Returns the active post lock.
 
-*Parameters*
+_Parameters_
 
- * state: Editor state.
+-   _state_ `Object`: Global application state.
 
-*Returns*
+_Returns_
 
-Whether content includes unsaved changes.
+-   `Object`: The lock object.
 
-### isEditedPostDirty
+<a name="getAdjacentBlockClientId" href="#getAdjacentBlockClientId">#</a> **getAdjacentBlockClientId**
 
-Returns true if there are unsaved values for the current edit session, or
-false if the editing state matches the saved or new post.
+Undocumented declaration.
 
-*Parameters*
+<a name="getAutosave" href="#getAutosave">#</a> **getAutosave**
 
- * state: Global application state.
+> **Deprecated** since 5.6. Callers should use the `getAutosave( postType, postId, userId )` selector from the '@wordpress/core-data' package.
 
-*Returns*
+Returns the current autosave, or null if one is not set (i.e. if the post
+has yet to be autosaved, or has been saved or published since the last
+autosave).
 
-Whether unsaved values exist.
+_Parameters_
 
-### isCleanNewPost
+-   _state_ `Object`: Editor state.
 
-Returns true if there are no unsaved values for the current edit session and
-if the currently edited post is new (has never been saved before).
+_Returns_
+
+-   `?Object`: Current autosave, if exists.
+
+<a name="getAutosaveAttribute" href="#getAutosaveAttribute">#</a> **getAutosaveAttribute**
+
+> **Deprecated** since 5.6. Callers should use the `getAutosave( postType, postId, userId )` selector from the '@wordpress/core-data' package and access properties on the returned autosave object using getPostRawValue.
+
+Returns an attribute value of the current autosave revision for a post, or
+null if there is no autosave for the post.
+
+_Parameters_
+
+-   _state_ `Object`: Global application state.
+-   _attributeName_ `string`: Autosave attribute name.
+
+_Returns_
+
+-   `*`: Autosave attribute value.
+
+<a name="getBlock" href="#getBlock">#</a> **getBlock**
```

Comment:
These undocumented symbols are the result of moving selectors/actions to core/block-editor. They shouldn't have been removed from the documentation. There is a fix for this at https://github.com/WordPress/gutenberg/pull/15424 to be merged before this one.

Reply:
15424 was merged.